GooglePlayInstant/Editor/PlayInstantIl2cppBuilder.cs

```diff
@@ -0,0 +1,75 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// A handler for building apks using quick deploy.
+    /// </summary>
+    public static class Il2cppBuilder
+    {
+        private const string SplashScenePath = "play-instant-loading-screen-scene.unity";
+        private static string _apkPathName = Path.GetFullPath("base.apk");
+
+        private static bool _projectIsUsingIl2Cpp =
+            PlayerSettings.GetScriptingBackend(BuildTargetGroup.Android) == ScriptingImplementation.IL2CPP;
+
+        /// <summary>
+        /// Whether or not the project is using IL2CPP as scripting backend. Ther return value of this is Updated by a
+        /// call to Il2cppBuilder.UpdateScriptingBackendInformation
+        /// </summary>
+        public static bool ProjectIsUsingIl2cpp
+        {
+            get { return _projectIsUsingIl2Cpp; }
+        }
+
+        /// <summary>
+        /// The specified path name of the target apk to be built.
+        /// </summary>
+        public static string ApkPathName
+        {
+            get { return _apkPathName; }
+            set
+            {
+                if (!Directory.Exists(Path.GetDirectoryName(Path.GetFullPath(value))))
+                {
+                    Debug.LogErrorFormat("Invalid File path. Directory {0} does not exist",
```

Comment:
Change the {0} to \"{0}\" in case it's not there. Nit: end 2nd sentence in a period.